README.md

```diff
@@ -112,6 +112,10 @@ This Project is part of the following programs :
 
 You can find our Code of Conduct [here](/CODE_OF_CONDUCT.md).
 
+## Music
+
+Music: https://www.bensound.com
```

Comment:
Also, You may remove the `Music` section from the `README` & put this link of reference in COMMENTS at the corresponding file in the Source Code(May be in the `Visual.html` at the end ), Saying something like `The music is taken from the link : ...`